docs/policies/recognizing_contributors_sponsors_and_partners.rst

```diff
@@ -55,7 +55,7 @@ cycle, for example, if the project is working on release 4.0, then a current con
 the Mautic Community on a regular basis, listed at :xref:`the Mautic Sponsor page`
 
 **Partner**: an organization who are financially and practically supporting the Mautic Community on a regular basis, and have met 
-the criteria to become a Mautic Community Partner, listed at :xref:`Mautic Community Partners` 
+the criteria to become a Mautic Community Partner, listed at :xref:`Mautic Partners` 
```

Comment:
Is there a reason for the line break here?

Reply:
Yes, I'm aware of the line breaks. They're still in the original state since they merged. I only fixed the link.

I plan to clean this up in separate PR as the changes don't belong in this PR. But I can fix them here, too, if you prefer. 

Just let me know!